### ISSUE TITLE
Analisar estimativas de de idade e gênero

### ISSUE THREAD

**Issue by vitalwarley** [vitalwarley/research#31]
https://paperswithcode.com/paper/mivolo-multi-input-transformer-for-age-and#code

**Comment by vitalwarley** [comment]
O [demo](https://github.com/wildchlamydia/mivolo#demo) do modelo acima foi bem fácil de executar.

**Comment by vitalwarley** [comment]
Consegui estimar idade e gênero das amostras de validação. Próximo passo é analisar via histogramas e afins, tanto individualmente, como em conjunto com os pares.

**Comment by vitalwarley** [comment]
Resultado parcial abaixo -- há algumas amostras onde o modelo não consegui detectar uma face (162/5045).

![image](https://github.com/vitalwarley/research/assets/6365065/5637a34f-f3db-462d-ab0d-2b464d377471)

**Comment by vitalwarley** [comment]
**Idade por face do par, por parentesco, para cada tipo de parentesco**

![image](https://github.com/vitalwarley/research/assets/6365065/4f0d3e7b-e47b-44a9-9a9e-8a1c10ee7246)

**Diferença na idade entre o par, por parentesco, para cada tipo de parentesco**

![image](https://github.com/vitalwarley/research/assets/6365065/50d21f5e-2a0b-4a33-b512-570db431badf)

**Gênero por face do par, por parentesco, para cada tipo de parentesco**

![image](https://github.com/vitalwarley/research/assets/6365065/e125888d-4cde-4448-8bf8-5f6160bedac8)

**Comment by vitalwarley** [comment]
_Since non-kin pairs are formed randomly from the true kin pair database itself, hence there is a significant probability that some non kin pairs have more similarity in feature space which is ideally undesirable and leads to improper learning of the model and results into errors._

[Easy pair selection method for Kinship Verification using fixed age group images](https://www.sciencedirect.com/science/article/abs/pii/S0262885623001014)

@tfvieira

**Comment by vitalwarley** [comment]
<details>
<summary>Diferença na idade entre o par, para cada tipo de parentesco (training set) </summary>

![image](https://github.com/vitalwarley/research/assets/6365065/06479f75-70be-407d-9b34-6ece6dd69c9d)

</details>

**Comment by vitalwarley** [comment]
<details>
<summary>Diferença na idade entre o par, para cada tipo de parentesco (test set) </summary>

![image](https://github.com/vitalwarley/research/assets/6365065/e0e1abe4-7d10-42ab-923d-579cd9007fc2)

</details>
